packages/browser/tsconfig.eslint.json

```diff
@@ -2,5 +2,5 @@
   "extends": "./tsconfig.json",
 
   "include": ["src/**/*"],
-  "exclude": []
+  "exclude": ["examples/**/*"]
```

Comment:
The examples are currently in JavaScript, not typescript, so should be excluded.